dumper-integration-tests/redshift-tests/src/main/java/com/google/edwmigration/dumper/csv/CsvUtil.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.edwmigration.dumper.csv;
+
+import static java.lang.Integer.parseInt;
+
+/** A helper class for reading and extracting data from CSV files. */
+public final class CsvUtil {
+
+  private CsvUtil() {}
+
+  /**
+   * @return String or an empty string if null.
+   */
+  public static String getStringNotNull(String value) {
+    return value == null ? "" : value;
+  }
+
+  /**
+   * @return int or 0 if "".
+   */
+  public static int getIntNotNull(String value) {
+    return getStringNotNull(value).equals("") ? 0 : parseInt(value);
+  }
```

Comment:
This is the same as `NumberUtils.toInt(value)` (`org.apache.commons.lang3.math.NumberUtils`)

Reply:
My bad, this is not exactly the same `NumberUtils` would return zero also in a case of a `NumberFormatException`, if the exception as to be thrown let's stay with current code.

Reply:
Returning zero in case of a NumberFormatException is Guava's Ints.tryParse()

Reply:
This method is no longer needed and it's removed, but I noted all three methods your guys suggested, thanks, good to know.

Done.